Sources/PointFree/Emails/Mailgun.swift

```diff
@@ -78,7 +77,7 @@ private func mailgunSend(email: Email) -> EitherIO<Error, Mailgun.SendEmailRespo
     |> \.allHTTPHeaderFields %~ attachedMailgunAuthorization
     |> \.httpBody .~ Data(urlFormEncode(value: params).utf8)
 
-  return jsonDataTask(with: request)
+  return jsonDataTask(with: request, logger: Current.logger)
```

Comment:
Making logger a dependency of these helpers, but we really should just add logging as a bracket around IO...maybe we can't.